WordPress/Classes/Networking/PlansFeaturesRemote.swift

```diff
@@ -82,23 +82,23 @@ class PlanFeaturesRemote: ServiceRemoteREST {
 
     private func fetchPlanFeatures(success: PlanFeatures -> Void, failure: ErrorType -> Void) {
         let endpoint = "plans/features"
-        let path = pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_2)
+        let path = pathForEndpoint(endpoint, withVersion: .Version_1_2)
         let locale = languageDatabase.deviceLanguage.slug
         let parameters = ["locale": locale]
 
-        api.GET(path,
+        wordPressComRestApi.GET(path,
                 parameters: parameters,
                 success: {
-                    [weak self] requestOperation, response in
+                    [weak self] response, httpResponse in
```

Comment:
I may have missed this in previous reviews, but for consistency with `SuccessResponseBlock` I think this should be `responseObject, httpResponse in`. Or in this case, since you aren't using the http response, `response, _ in`